conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -13,53 +13,54 @@
 
 #Pretends that that journey starts from Bta
-GET  /set-up-a-payment-plan/test-only/start-journey-epaye-bta     testOnly.controllers.StartJourneyController.startJourneyEpayeBta
-GET  /set-up-a-payment-plan/test-only/start-journey-epaye-epaye   testOnly.controllers.StartJourneyController.startJourneyEpayeEpaye
+GET  /set-up-a-payment-plan/test-only/start-journey-epaye-bta           testOnly.controllers.StartJourneyController.startJourneyEpayeBta
+GET  /set-up-a-payment-plan/test-only/start-journey-epaye-epaye         testOnly.controllers.StartJourneyController.startJourneyEpayeEpaye
 
-GET  /set-up-a-payment-plan/test-only/start-journey-vat-bta       testOnly.controllers.StartJourneyController.startJourneyVatBta
-GET  /set-up-a-payment-plan/test-only/start-journey-vat-vat       testOnly.controllers.StartJourneyController.startJourneyVatVatService
-GET  /set-up-a-payment-plan/test-only/start-journey-vat-penalties testOnly.controllers.StartJourneyController.startJourneyVatVatPenalties
+GET  /set-up-a-payment-plan/test-only/start-journey-vat-bta             testOnly.controllers.StartJourneyController.startJourneyVatBta
+GET  /set-up-a-payment-plan/test-only/start-journey-vat-vat             testOnly.controllers.StartJourneyController.startJourneyVatVatService
+GET  /set-up-a-payment-plan/test-only/start-journey-vat-penalties       testOnly.controllers.StartJourneyController.startJourneyVatVatPenalties
 
-GET  /set-up-a-payment-plan/test-only/start-journey-sa-bta        testOnly.controllers.StartJourneyController.startJourneySaBta
-GET  /set-up-a-payment-plan/test-only/start-journey-sa-pta        testOnly.controllers.StartJourneyController.startJourneySaPta
-GET  /set-up-a-payment-plan/test-only/start-journey-sa-mobile     testOnly.controllers.StartJourneyController.startJourneySaMobile
+GET  /set-up-a-payment-plan/test-only/start-journey-sa-bta              testOnly.controllers.StartJourneyController.startJourneySaBta
+GET  /set-up-a-payment-plan/test-only/start-journey-sa-pta              testOnly.controllers.StartJourneyController.startJourneySaPta
+GET  /set-up-a-payment-plan/test-only/start-journey-sa-mobile           testOnly.controllers.StartJourneyController.startJourneySaMobile
 
 
-GET  /set-up-a-payment-plan/test-only/pega                         testOnly.controllers.PegaController.dummyPegaPage(regime: TaxRegime)
-POST /set-up-a-payment-plan/test-only/pega                         testOnly.controllers.PegaController.dummyPegaPageContinue(regime: TaxRegime)
+GET  /set-up-a-payment-plan/test-only/pega                              testOnly.controllers.PegaController.dummyPegaPage(regime: TaxRegime)
+POST /set-up-a-payment-plan/test-only/pega                              testOnly.controllers.PegaController.dummyPegaPageContinue(regime: TaxRegime)
+GET  /set-up-a-payment-plan/test-only/paying-within-six-months/back     testOnly.controllers.PegaController.backFromPegaLanding(regime: TaxRegime)
```

Comment:
this would be more consistent with the other pega endpoints we have:
```suggestion
GET  /set-up-a-payment-plan/test-only/pega/back     testOnly.controllers.PegaController.backFromPegaLanding(regime: TaxRegime)
```